tests/ats/test_fixtures.py

```diff
@@ -21,6 +21,15 @@ def fixtures(kube_cluster: Cluster):
     ret = kube_cluster.kubectl("apply", filename="cluster-crd.yaml", output_format="json")
     LOGGER.debug("Created cluster CRD")
 
+    # Organization CRD
+    LOGGER.info("Create Organization CRD")
+    ret = kube_cluster.kubectl("apply", filename="https://raw.githubusercontent.com/giantswarm/organization-operator/main/config/crd/security.giantswarm.io_organizations.yaml", output_format="json")
+    LOGGER.debug(f"Created Organization CRD: {ret}")
+    ret = kube_cluster.kubectl(f"patch organization giantswarm --subresource status --patch-file organization-giantswarm-patch.yaml")
```

Comment:
This patch fails in the test. I can see the CRD being applied here but does the org giantswarm cr exist at this point?

Reply:
You are right. I moved the patch to later down in the fixture, but tests still fail.